go.mod

```diff
@@ -39,5 +38,5 @@ require (
 	k8s.io/apimachinery v0.18.8
 	k8s.io/client-go v0.18.8
 	sigs.k8s.io/controller-runtime v0.6.2
-	sigs.k8s.io/controller-tools v0.2.4
+	sigs.k8s.io/controller-tools v0.4.0
```

Comment:
@muvaf this is bumping all CRDs to v1 because v0.4.0 uses v1 by default. Is it your intention to make that change in this PR?

Reply:
No, I had to update to `v0.4.0` because that's the minimum supported by ACK-generated CRDs.